src/Terrain/TerrainType.java

```diff
@@ -90,14 +90,14 @@ private TerrainType(int attributeFlags, int defense, Color mainColor, String nam
   private static final String TEMP_AIRPORT_NAME = "TEMP_AIRPORT";
   public static final TerrainType TEMP_AIRPORT = new TerrainType( TEMP_AIRPORT_FLAGS, TEMP_AIRPORT_DEFENSE, TEMP_AIRPORT_COLOR, TEMP_AIRPORT_NAME, GRASS );
 
-  private static final int BRIDGE_FLAGS = LAND | WATER;
+  private static final int BRIDGE_FLAGS = LAND;
```

Comment:
But... bridge terrains do have water. Does this mean ships can't go there now?

Reply:
Correct. That's how AW2 works.
DoR-style bridges are definitely interesting, but most of the map design we have now assumes AW2 bridges anyway.